StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1645IncludedDocumentationFileDoesNotExist.cs

```diff
@@ -30,6 +30,7 @@
     /// loaded.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:question: Would it be reasonable to provide a code fix that locates a documentation file with a matching name at a different location in the project, and offers to fix the path to point to it? With the work towards running .NET on Linux, I wonder if such a code fix would also be useful to catch unintentional case differences between the file on disk and the `<include>` element.